CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Snyk Changelog
 
+## [2.7.7]
+### Fixed
+- (LS Preview) do not hang on missing answers to message requests, timeout after 5s
```

Comment:
What does this `(LS Preview)` communicate? Is it the same as the label `(pre-alpha)` added for `[2.7.5]`

Reply:
Yeah, it's just marking, that this is for the preview. Before it was pre-alpha, now we're taking step by step more people on.